src/fragments/lib-v1/storage/native_common/configureaccess/common.mdx

```diff
@@ -30,9 +30,9 @@ import flutter2 from '/src/fragments/lib-v1/storage/flutter/configureaccess/10_p
 
 <Fragments fragments={{ flutter: flutter2 }} />
 
-This will upload with the prefix `/protected/[IDENTITY_ID]/` followed by the key.
+This will upload with the prefix `/protected/[IDENTITY_ID]/` followed by the `key`.
 
-For other users to read the file, you must specify the user ID of the creating user in the passed options.
+For other users to read the file, you must specify the access level as `protected` and the identity ID of the user who uploaded it in the options.
```

Comment:
should this be reverted?

Reply:
It isn't really in scope but seems to be an improvement as it is more specific. IMO it is fine to leave it but if you want to remove it to reduce scope and PR size that is fine too.